src/logic/operate.js

```diff
@@ -0,0 +1,43 @@
+/*eslint-disable*/
```

Comment:
Is there a reason why the whole eslint is disabled for this file? :thinking:

Reply:
It was unintended. Sorry for this.